File: IamStupendous.Resume/src/IamStupendous.Resume/wwwroot/scripts/app/skill/skill.ts
/**
 * Skill model class
 */
export class Skill {
    name: string;
    expertise: number;
    unknown: number;

    constructor(name: string,
                expertise: number) {
        this.name = name;
        this.expertise = expertise;
        this.unknown = 100 - this.expertise;
    }
}

File: IamStupendous.Resume/src/IamStupendous.Resume/Models/Skill.cs
namespace IamStupendous.Resume.Models
{
    public class Skill
    {
        public string Name { get; set; }
        public int Expertise { get; set; }

        public Skill(string name, int expertise)
        {
            Name = name;
            Expertise = expertise;
        }
    }
}